### PR TITLE
removed unused imports

### PR DESCRIPTION
**Pull request by munishchouhan**
This PR will do the following:
1. Remove unused imports in build request class
2. ~~remove .trivy-cache in build ci~~

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -72,9 +72,9 @@ jobs:
           AZURECR_PAT: ${{ secrets.AZURECR_PAT }}
           GOOGLECR_KEYS: ${{ secrets.GOOGLECR_KEYS }}
 
-      - name: Cleanup build workspace
+      - name: Cleanup trivy cache
         run: |
-          sudo rm -rf /home/runner/work/wave/wave/build-workspace
+          sudo rm -rf /home/runner/work/wave/wave/build-workspace/.trivy-cache
```
Comment (pditommaso) [inline .github/workflows/build.yml:77]: Is this problem still around? why only .trivy-cache ?

Comment (munishchouhan) [reply .github/workflows/build.yml:77]: yes, the problem is still around, I think in the previous command its not delete the hidden folders, and if i use .*, it does not execute, the problem is related to the access of `.trivy-cache`. So i tried it in this PR https://github.com/seqeralabs/wave/pull/503 and ci start working 
Screenshot 2024-05-23 at 15 54 11

Comment (munishchouhan) [reply .github/workflows/build.yml:77]: previous tries

1. with `sudo rm -rf /home/runner/work/wave/wave/build-workspace/.*`
Screenshot 2024-05-23 at 15 56 57

3. with `sudo rm -rf /home/runner/work/wave/wave`
Screenshot 2024-05-23 at 15 57 19

Comment (pditommaso) [reply .github/workflows/build.yml:77]: I'm quite sure `rm -rf /foo/bar` remove ALL bar content and the directory itself. therefore it would solve

Comment (munishchouhan) [reply .github/workflows/build.yml:77]: you are right, the problem is github ci skip this delete step in some failures, i am not sure why 
as happened in this run
https://github.com/seqeralabs/wave/actions/runs/9206439473/job/25324299024

Comment (pditommaso) [reply .github/workflows/build.yml:77]: Likely [this](https://github.com/seqeralabs/wave/commit/8adb52ab355dd72ff3d4d19a51d42a63fdc2e546) should solve

**Pull request merged**